src/webgpu/shader/validation/wgsl/function-return-missing-return-empty-body.fail.wgsl

```diff
@@ -0,0 +1,9 @@
+# v-0002 - This program fails due to the missing return at the end of the
+# function.
```

Comment:
This test case seems almost identical to the one right below it.

Reply:
In this one main() is empty and func() is not called anywhere.